app/src/main/java/com/grindrplus/Hooker.kt

```diff
@@ -15,7 +15,12 @@ class Hooker : IXposedHookLoadPackage {
     fun toastInvalidVersionName() {
         Toast.makeText(
             appContext,
-            "This hook is for client version $TARGET_PKG_VERSION_NAME. (Current: $pkgVersionName) Hook will not be loaded.",
```

Comment:
Does this show the toasts one after another?

Reply:
So far as the result of my tests, it will show toasts one after another. But I still try to find a way to make a toast large enough to show the whole text.